cloudify/workflows/tasks.py

```diff
@@ -540,6 +541,8 @@ def _set_queue_kwargs(self):
                 self._task_tenant = tenant
         self.kwargs['__cloudify_context']['task_queue'] = self._task_queue
         self.kwargs['__cloudify_context']['task_target'] = self._task_target
+        if rest_host:
+            self.kwargs['__cloudify_context']['rest_host'] = rest_host
```

Comment:
OK so let's just set this ONLY here,
and not here and in operation_context

so: in the `else` branch below (in the MGMTWORKER_QUEUE) branch, instead of None, return what is now returned in operation_context, ie `self.workflow_ctx.rest_host`

Reply:
Sure, make sense will update that